### DISCUSSION TITLE
False detection?

### DISCUSSION THREAD

**Discussion by xenoAVP** [Tyrrrz/DiscordChatExporter#982] [General]
Hey! Haven't used DiscordChatExporter in a while, I just came back to it so there was an update, just after DCE finished downloading the necessary Dotnet runtime installer I ran an antivirus scan and a few detections popped up:
![image](https://user-images.githubusercontent.com/54017343/211161887-02340c7e-ca3a-4e62-85a9-f1ad68ce6eb6.png)
Are these false positives or should I be worried?

**Comment by xenoAVP** [comment]
![image](https://user-images.githubusercontent.com/54017343/211162158-af0f0ebd-0332-4be6-b073-dbd507f6c1dd.png)
![image](https://user-images.githubusercontent.com/54017343/211162164-08fbd9d6-8046-4f58-8319-00f2a4f33b77.png)
![image](https://user-images.githubusercontent.com/54017343/211162190-b6b8dce0-b241-43a7-abde-ac117f543a4b.png)
![image](https://user-images.githubusercontent.com/54017343/211162200-c2f8ff65-33a2-4cb1-9188-f31f5c704c47.png)
Here's a more detailed description of the threats that were found

**Reply by Tyrrrz** [reply]
It shouldn't download any `cmd` files though, so that's kind of weird. Can you access the content of one of those suspicious files and post it here?

**Reply by xenoAVP** [reply]
Yep, sorry, It was a false detection and has nothing to do with DiscordChatExporter. I coincidentally ran a KVRT scan before running NPE, which created that file, it's a kaspery cleanup script...
